tests/runtests.sh

```diff
@@ -90,7 +95,7 @@ function fail () {
 }
 
 function start_toaster() {
-    mkdir $tempdir/toasterbuild
+    mkdir -p $tempdir/$toasterbuild
```

Comment:
Why is the -p added? It was partly intended to make sure it wasn't trying to create something off in the weeds where not expected.